src/common/variables.c

```diff
@@ -146,16 +160,37 @@ static void init_expansion(dt_variables_params_t *params, gboolean iterate)
 
     params->data->flags = img->flags;
 
-    params->data->max_height = img->p_height;
-    params->data->max_width = img->p_width;
+    params->data->max_height = dt_conf_get_int(CONFIG_PREFIX "height");
```

Comment:
I don't like having this coming from the prefs. Those values can be changed from export module or using a preset in export module. So to me we are probably introducing a race condition here.